### DISCUSSION TITLE
Code 404 error in stop-publication-esentool method in the preview environment.

### DISCUSSION THREAD

**Discussion by PilarMoralesViewnext** [OP-TED/eForms-SDK#657] [Q&A] [answered]
Hi there,
We are getting a 404 error when we try to stop a notice that is in the Submitted state in the preview environment..

![image](https://github.com/OP-TED/eForms-SDK/assets/107406639/b4b96e3c-3638-47df-853f-c636210e6e3a)
![image](https://github.com/OP-TED/eForms-SDK/assets/107406639/8aa888f8-f535-48bd-bf4e-3683349458e4)

Has anything changed from previous versions that we should take into consideration? These same calls have worked for us before.

Thanks in advanced

**Comment by pcolast** [comment]
Dear user,

Due to a huge number of submitted notices there has been a delay in their processing until they have been stored in the TEDMON22 system.

It should be possible to stop publication of your notice now. Could you please try ?

Since your API key has been disclosed, we recommend that you generate a new one. 

Thank you.

Have a nice day,
Stefan

**Comment by PilarMoralesViewnext** [comment]
Hi again,
We tried again this morning and the notice still in SUBMITTED state, at least that is what the search method confirm, but the response of the stop method is now:

![image](https://github.com/OP-TED/eForms-SDK/assets/107406639/8ac65ef2-8c5c-4e97-b97a-3719a7fd44e8)

Will be usual the delay in the processing? To prepare our system for this case.

Thanks,

Happy thursday,

Pilar

**Reply by pcolast** [reply]
Dear Pilar,

Publishing status is not yet exposed, therefore some notices cannot be stopped even though they have status SUBMITTED.
When you tried the stop publication, it seems it was for another notice that was in status publishing. It should be possible to stop the notice b70b4c97-9ad0-4f36-85ea-b0427742af7b-01 that you originally reported.

We are currently experiencing some delays in the processing; we are working to fix the issues as soon as possible. 
In the meantime, in case the stop doesn't work for b70b4c97-9ad0-4f36-85ea-b0427742af7b-01, please provide us with a full screenshot of the error containing time and date of the request, so that we can look into it further.

Thank you,
Stefan

**Reply by PilarMoralesViewnext** [reply]
Hello Stefan,
Thank you for your answer.
True, We was able to stop the notice: b70b4c97-9ad0-4f36-85ea-b0427742af7b-01.
Sorry for the misunderstanding, yesterday we tested many notices with the same error in stop method and the one I put in the previous comment is: 0daf00f0-bfa8-4f53-8f9c-588fad36ca73-01.

In any case, all this "delays" will help us to improve our system for future issues.

Will the Publishing status be exposed in any time soon? 

Pilar

**Reply by pcolast** [reply, marked as answer]
Dear Pilar,

Publishing status will be exposed in the next version ( v1.11) of eNotices2.

Have a nice day,
Stefan
